### PR TITLE
Added the index.html and styles.css files

### PR DESCRIPTION
**Pull request by chiedozie013**
In this project I:
- created the index.html file,
- created the forms on the index.html page,
- created the styles.css page,
- styled the form in the styles.css page

### CODE REVIEW

**Review by chiedozie013 — COMMENTED** [review]
1. This is a good work. You've done really well and it shows you put in a lot of effort but I think your choice of color should be looked into. Try another color for the submit button.
2. You obviously did not use linters check for this project. please include it so it would check and fix your project professionally.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @chiedozie013, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job reviewing your project :+1: 
- Great job phrasing your comment is politely as suggestions instead of orders :1st_place_medal: 

## Required Changes ♻️
- [ ] Nice work reviewing your project :clap: However It would be nice to add at least two(02) comments for the review and use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments to follow the project requirements. Watch this [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Kindly consider using a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

- [ ] In future projects, kindly make sure your branch name has a feature or action name which determines what is in the branch.
  - **Examples of a good branch name: setup_mobile_first, review-trials-project**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  display: block;
+  width: 100%;
+  background: #13b6cb;
+  border-radius: 8px;
+  cursor: pointer;
+  height: 3rem;
+  color: #ffffff;
+}
+.container {
+  /* width:100%; */
```
Comment (chiedozie013) [inline styles.css:97]: Good job! Please do not leave unused codes as comments on your codebase. they don't make your code look neat and easily readable.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (chiedozie013) [inline index.html:23]: Again, do not leave unused code as comments on your codebase. Please always ensure to have a clean and readable codebase

**Review by chiedozie013 — COMMENTED** [review]
# changes required ♻️ 

Please take a look at my comments below and make all the necessary changes.

submit for re-review when you are done effecting all changes.

thanks and all the best 👍

**Review by usorfaitheloho — APPROVED** [review]
Hi @chiedozie013  , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**